core/domain/exp_domain_test.py

```diff
@@ -7856,7 +7985,7 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
 title: Title
 """)
 
-    YAML_CONTENT_V37_WITH_IMAGE_CAPTION = ("""author_notes: ''
```

Comment:
Ditto

Reply:
@aks681 I also found that the YAML_CONTENT_V37_WITH_IMAGE_CAPTION is repeated twice. So I think it maybe is a mistake.

Reply:
@aks681 I have removed one of the duplicate yaml content.